### PR TITLE
added new events, group and display_date, and everything is cards now

### PR DESCRIPTION
**Pull request by wendko**

### CODE REVIEW

**Comment by wendko** [comment]
image

File: data/meetups.yml [hunk 1]
```diff
@@ -3,47 +3,115 @@ template:
...
+
+tbc: &tbc
+  venue: "Venue to be confirmed"
 
 list:
 - name: "RubyConf MY 2018"
   map: "https://goo.gl/maps/eMeuapNw36L2"
   address: "Menara Ta One, KLCC 22, Jalan P. Ramlee, Kuala Lumpur"
   venue: "One Auditorium"
+  display_date: Thu 25 - 26 October 2018, 8:00am - 5:00pm
```
Comment (parasquid) [inline data/meetups.yml:45]: I was actually thinking of using javascript to read the original date and format it for human readability to reduce duplication in the source files.

Do you want to try that before merging?

Comment (parasquid) [reply data/meetups.yml:45]: Here's what I'm thinking of:
```
date: 2018-10-25T08:00:00+08:00
until: 2018-10-26T17:00:00+08:00
```

then we do the `dates` key to be parsed by js and autogenerated.

Comment (parasquid) [reply data/meetups.yml:45]: Okay, I think we'll merge it first and I can just do a simple implementation for this one before the data file becomes too large :P

Comment (wendko) [reply data/meetups.yml:45]: Sure! I'm all for using javascript, it took a while to format the dates. Easy to make mistakes also. I'd need to learn how to do that though.

Comment (wendko) [reply data/meetups.yml:45]: Also feel free to remove the events that you've mentioned earlier.

**Pull request merged**